tempest/api/workloadmgr/snapshot/test_snapshot_mount.py

```diff
@@ -93,3 +93,137 @@ def test_1_filesearch_default_parameters(self):
             LOG.error("Exception: " + str(e))
             reporting.set_test_script_status(tvaultconf.FAIL)
             reporting.test_case_to_write()
+
+
+    @test.attr(type='smoke')
+    @test.idempotent_id('28fd0710-ef00-42b0-93f0-9dbb3ffc5bee')
+    def test_2_umount_snapshot(self):
+        reporting.add_test_script(str(__name__) + "_umount_snapshot")
+        try:
+            if self.exception != "":
+                LOG.debug("pre req failed")
+                reporting.add_test_step(str(self.exception), tvaultconf.FAIL)
+                raise Exception (str(self.exception))
+            LOG.debug("pre req completed")
+
+            global instances_ids
+            global snapshot_ids
+            global wid
+            global security_group_id
+            global volumes_ids
+            global fvm_id
+            global floating_ips_list
+            instances_ids = self.instances_ids
+            snapshot_ids = self.snapshot_ids
+            wid = self.wid
+            volumes_ids = self.volumes_ids
+            security_group_id = self.security_group_id
+            fvm_id = self.fvm_id
+            dismount_snapshot_id = snapshot_ids[0]
+            floating_ips_list = self.floating_ips_list
+
+            LOG.debug("dismount snasphot of a  snapshot")
+            is_dismounted = self.dismount_snapshot(dismount_snapshot_id)
```

Comment:
There is no dismount_snapshot method in base.py. Please check.

Reply:
commited function